azure/src/test/scala/quasar/blobstore/azure/AzurePropsServiceSpec.scala

```diff
@@ -91,12 +91,5 @@ class AzurePropsServiceSpec extends Specification with CatsIO {
         BlobPath(List(PathElem("something"))),
         beNone)
     }
-
-    "invalid container returns none" >> {
```

Comment:
Why was this and the other test removed?

Reply:
The service cannot be created from this config anymore. 
Was planning to replace them with something else before merge though, if possible.